### DISCUSSION TITLE
Multiple RDP session in windows 10 or windows 7

### DISCUSSION THREAD

**Discussion by idforfast** [kimmknight/remoteapptool#109] [General]
![Screenshot 2024-01-23 173650](https://github.com/kimmknight/remoteapptool/assets/140324774/e78e5a3e-7ab1-4c59-b309-095b58e13022)
Is it possible to activate multiple Remote Desktop Protocol (RDP) sessions on a Windows 10 or Windows 7 client machine using this software? I'm attempting it on Windows 10, but it prompts me to log off the current user before logging in. I'm seeking a solution that allows for multiple sessions without requiring the logoff and login process.

**Comment by MrBrianGale** [comment]
This tool is more of a front end for Microsoft Remote Apps. It is the same protocol, so it has the same limitations. Remote Apps (part of the Remote Desktop protocol) requires a user session and desktop OS's like Windows 10 and 7 do not allow for multi-user environments. If you need multi-user, you must use a Server based OS OR use some third party tool s to allow for multi-user on single user machines. Offhand, I am not certain if those tools result in violations of licenses.
This tool doesn't enable multi-user on single user systems.
